exporter/collector/metrics.go

```diff
@@ -179,12 +179,12 @@ func (me *MetricsExporter) PushMetrics(ctx context.Context, m pdata.Metrics) err
 	for i := 0; i < rms.Len(); i++ {
 		rm := rms.At(i)
 		monitoredResource, extraResourceLabels := me.mapper.resourceToMonitoredResource(rm.Resource())
-		ilms := rm.InstrumentationLibraryMetrics()
+		ilms := rm.ScopeMetrics()
```

Comment:
same nit here, `sms` or similar may be more appropriate now and `s/ilm/sm` below?

Reply:
done